src/index.js

```diff
@@ -0,0 +1,71 @@
+import './style.css';
+
+const grab = (e) => document.getElementById(e);
+const sub = grab('sub');
+const refBtn = grab('ref-btn');
+const ol = grab('list');
+
+const refresh = async (gameID) => {
+  await fetch(`https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${gameID}/scores/`)
+    .then((response) => {
+      const res = response.json();
+      res.then((finalResponse) => {
+        const { result } = finalResponse;
+        result.forEach((element) => {
+          ol.innerHTML += `<li>${element.user}:${element.score}</li>`;
+        });
+      });
+    })
+    .catch((error) => { throw new Error(error); });
+};
+
+refBtn.addEventListener('click', async () => {
+  const { result } = JSON.parse(localStorage.getItem('gameID') || '');
+  const newID = result.slice(14, 34);
+  if (newID !== null) {
+    ol.innerHTML = ''; // clear out elements before calling the refresh
+    refresh(newID);
+  }
+});
+
+const init = async () => {
+  if (localStorage.getItem('gameID') === null) {
+    fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/', {
+      method: 'POST',
+      body: JSON.stringify({
+        name: 'Formula1',
+      }),
+      headers: {
+        'Content-type': 'application/json; charset=UTF-8',
+      },
+    })
+      .then((response) => response.json())
+      .then((data) => {
+        localStorage.setItem('gameID', JSON.stringify(data));
+      });
+  }
```

Comment:
- [ ] Use `async `and `await `with the fetch method to send and receive data. To reproduce the `then ` step in your code you can store each step result in a variable using the await keyword and use the previous result in the next call. eg: 
```js
const response = await  fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/', {
      method: 'POST',
      body: JSON.stringify({
        name: 'Formula1',
      }),
      headers: {
        'Content-type': 'application/json; charset=UTF-8',
      },
    })

const data = await response.json()
localStorage.setItem('gameID', JSON.stringify(data));
```

Reply:
done